### DISCUSSION TITLE
Erro de url da API ao postar pacote

### DISCUSSION THREAD

**Discussion by JotaPeCarvalho** [balta-io/forum#232] [Tira Dúvidas] [answered]
Olá, está aparecendo esse erro pra mim, já tentei várias urls e não foi
![image](https://user-images.githubusercontent.com/91575096/213253946-1828ceef-eb54-4e69-8049-e39432d180ac.png)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado:
https://github.com/balta-io/forum/discussions/223
